modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustClientCodegen.java

```diff
@@ -301,9 +301,6 @@ public void processOpts() {
         if (HYPER_LIBRARY.equals(getLibrary())) {
             supportingFiles.add(new SupportingFile("request.rs", apiFolder, "request.rs"));
         }
```

Comment:
I'm afraid your change breaks `library: hyper`, as can easily be checked like this:
```
java -jar ./modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -g rust -i https://github.com/slackapi/slack-api-specs/raw/master/web-api/slack_web_openapi_v2.json -o slack --skip-validate-spec --remove-operation-id-prefix --library hyper
cargo check --manifest-path slack/Cargo.toml
> ...
> error[E0583]: file not found for module `client`
>    --> src\apis\mod.rs:162:1
>     |
> 162 | pub mod client;
>     | ^^^^^^^^^^^^^^^
>     |
>     = help: to create the module `client`, create file "src\apis\client.rs"
```
I would suggest something like this as an alternative for removing the line completely:

```suggestion
        if (HYPER_LIBRARY.equals(getLibrary())) {
            supportingFiles.add(new SupportingFile("request.rs", apiFolder, "request.rs"));
            supportingFiles.add(new SupportingFile(getLibrary() + "/client.mustache", apiFolder, "client.rs"));
        }
```

Maybe a test using `library: hyper` would be helpful to prevent something like this in the future?

Reply:
Good catch! I didn't think that client file was shared between hyper and reqwest. Will fix that. :+1: